lib/libinput/BUILD

```diff
@@ -1,11 +1,5 @@
 OPTS+=" --prefix=/usr \
-        -Ddebug-ui=false \
-        -Dtests=false"
+        -D debug-ui=false \
```

Comment:
According to meson docs options needs to be in the form -Dopt=value, not -D opt=value. @Florin65 Could you revert those changes please ?